<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-IM3ORJ1K" failures="1" tests="2" name="Banking App Test" time="38.279" errors="0" timestamp="15 Jan 2022 13:36:05 GMT">
  <testcase classname="com.testCases.TC_02_Login_DDT" name="loginDDT" time="11.23"/>
  <testcase classname="com.testCases.TC_02_Login_DDT" name="loginDDT" time="3.684">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at com.testCases.TC_02_Login_DDT.loginDDT(TC_02_Login_DDT.java:29)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
</testsuite> <!-- Banking App Test -->
